Cannot query 'show tags'

```
Caused by: java.sql.SQLException: Method executeQuery() can only execute nGql to query data, but the current nGql do not contains any query keyword in [match, lookup, go, fetch, find, subgraph], please modify your nGql or use executeUpdate(), execute().
	at com.vesoft.nebula.jdbc.statement.NebulaStatementImpl.checkReadOnly(NebulaStatementImpl.java:97)
	at com.vesoft.nebula.jdbc.statement.NebulaPreparedStatementImpl.executeQuery(NebulaPreparedStatementImpl.java:44)
	at org.springframework.jdbc.core.JdbcTemplate$1.doInPreparedStatement(JdbcTemplate.java:722)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:651)
```

--- a/src/main/java/com/vesoft/nebula/jdbc/statement/NebulaStatementImpl.java
+++ b/src/main/java/com/vesoft/nebula/jdbc/statement/NebulaStatementImpl.java
@@ -19,7 +19,7 @@ import org.slf4j.LoggerFactory;
 public class NebulaStatementImpl implements NebulaStatement {
     protected final Logger log = LoggerFactory.getLogger(this.getClass());
     protected final String[] updateKeyword = {"update", "delete", "insert", "upsert", "create", "drop", "alter", "rebuild"};
-    protected final String[] queryKeyword = {"match", "lookup", "go", "fetch", "find", "subgraph"};
+    protected final String[] queryKeyword = {"match", "lookup", "go", "fetch", "find", "subgraph", "show"};
 
     protected String nGql;
     protected NebulaConnection nebulaConnection;

--- a/src/main/java/com/vesoft/nebula/jdbc/statement/NebulaStatementImpl.java
+++ b/src/main/java/com/vesoft/nebula/jdbc/statement/NebulaStatementImpl.java
@@ -19,7 +19,7 @@ import org.slf4j.LoggerFactory;
 public class NebulaStatementImpl implements NebulaStatement {
     protected final Logger log = LoggerFactory.getLogger(this.getClass());
     protected final String[] updateKeyword = {"update", "delete", "insert", "upsert", "create", "drop", "alter", "rebuild"};
-    protected final String[] queryKeyword = {"match", "lookup", "go", "fetch", "find", "subgraph", "show"};
+    protected final String[] queryKeyword = {"match", "lookup", "go", "fetch", "find", "subgraph", "show", "desc", "describe"};
 
     protected String nGql;
     protected NebulaConnection nebulaConnection;

--- a/src/test/java/NebulaStatementTest.java
+++ b/src/test/java/NebulaStatementTest.java
@@ -4,6 +4,7 @@
  */
 
 import com.vesoft.nebula.jdbc.NebulaDriver;
+import com.vesoft.nebula.jdbc.statement.NebulaStatementImpl;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
@@ -112,5 +113,8 @@ public class NebulaStatementTest {
     }
 
 
-
+    @Test
+    void testCheckReadOnlyShowQuery() throws SQLException {
+        new NebulaStatementImpl(null).checkReadOnly("show tags");
+    }
 }

